charts/brig/values.yaml

```
@@ -69,6 +69,8 @@ config:
     setDeleteThrottleMillis: 100
     # Allow search within same team only. Default: false
     # setSearchSameTeamOnly: false|true
+    # Allow settings max number of user clients. Default: 7
```

```suggestion
    # Set max number of user clients. Default: 7
```
Slightly weird wording, this flag does not really allow setting a max number, but just sets it, right?

---

Yes, thank you!